app/controllers/components/course/lectures_component.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+class Course::LecturesComponent < SimpleDelegator
+  include Course::ControllerComponentHost::Component
+
+  def self.display_name
+    I18n.t('components.lectures.name')
+  end
+
+  def sidebar_items
+    main_sidebar_items + settings_sidebar_items
+  end
+
+  def self.enabled_by_default?
+    false
+  end
+
+  def settings
+    @settings ||= Course::LectureSettings.new(current_course.settings(:lecture))
+  end
+
+  private
+
+  def main_sidebar_items
+    [
+      {
+        key: :lectures,
+        icon: 'television',
+        title: settings.title || t('course.lectures.sidebar_title'),
+        weight: 1,
```

Comment:
You might want to compare this weight with the rest - do you want lectures to be at the top? By default we usually leave Announcements at the top.